README.md

```diff
@@ -1 +1,134 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Todo List <a name="about-project"></a>
+
+**Todo List** is a tool for storing all your taks...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Build With</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Store your tasks**
+- **Mark your tasks as completed**
+- **Order your tasks by priority**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon](https://github.com/anthonymr/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need: a properly working pc or laptop and a modern web browser.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/anthonymr/Todo-list.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-folder
+  git clone https://github.com/anthonymr/Todo-list.git
+```
+
+### Usage
+
+To run the project, open index.html in your web browser.
+
+### Run tests
+
+To run tests, run the following command: Right click on the index.html document then open with live server. Make sure live server extension is install in your text editor.
+
+### Deployment
+
+You can deploy this project using: GitHub but if you want to deploy make sure you let me know first :3
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Anthony Martin**
+
+- GitHub: [Anthony Martin](https://github.com/yash244466666)
+- LinkedIn: [Anthony Martin](https://www.linkedin.com/in/anthony-martin-8820a3117/)
+- Twitter: [Anthony Martin](https://twitter.com/Anthony2Martin)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Upgraded style**
+- [ ] **Live demo**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please, share it with your friends.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [Optional] License file contains obsolete information which is not professional. The date and type of app need to be edited